### PR TITLE
Portuguese brazil

### PR DESCRIPTION
**Pull request by luciormoraes**
Translating the following files:
- embedding.article
- exporting.article
- generics-struct-types.article

### CODE REVIEW

File: _content/tour/por/generics-struct-types.article [hunk 1]
```diff
@@ -71,23 +71,23 @@ declared as pointers to a node of some type T using the square bracket syntax.
         return &n
     }
 
-This is an implementation of a method named add for the list type. No formal generic type
-list declaration is required (as with functions) since the method is bound to the list through
-the receiver. The add method’s receiver is declared as a pointer to a list of some type T and
-the return is declared as a pointer to a node of the same type T.
+Esta é uma implementação de um método chamado add para o tipo list. Nao e necessario a declaracao de nenhum tipo generico
```
Comment (rschio) [inline _content/tour/por/generics-struct-types.article:74]:
```suggestion
Esta é uma implementação de um método chamado add para o tipo list. Não é necessária a declaração de nenhum tipo genérico 
```

File: _content/tour/por/generics-struct-types.article [hunk 2]
```diff
@@ -71,23 +71,23 @@ declared as pointers to a node of some type T using the square bracket syntax.
...
-Without generics, this entire method would need to be duplicated since the
-construction of a node would need to be hard coded to a known, declared type prior
-to compilation. Since the amount of code (for the entire list implementation) that
-needs to change for different data types is very small, being able to declare a node
-and list to manage data of some type T reduces the cost of code duplication and
-maintenance.
+Sem generics, todo este método precisaria ser duplicado, uma vez que o
+a construção de um node precisaria ser codificada para um tipo conhecido e declarado antes
+para compilação. Como a quantidade de código (para toda a implementação da lista) que
+precisa mudar para diferentes tipos de dados é muito pequeno, sendo capaz de declarar um nó
```
Comment (rschio) [inline _content/tour/por/generics-struct-types.article:88]:
```suggestion
para compilação. Como a quantidade de código (para toda a implementação da lista) que
precisa mudar para diferentes tipos de dados é muito pequena, sendo capaz de declarar um nó
```

**Review by rschio — COMMENTED** [review]
Very good! Just some minor things.

File: _content/tour/por/exporting.article [hunk 3]
```diff
@@ -85,11 +83,11 @@ should be of an exported type.
...
-When it comes to fields in a struct, the first letter declares if the field is
-accessible to code outside of the package it’s declared in. In this case, Name
-and ID are accessible, but password is not. It’s an idiom to separate exported
-and unexported fields in this manner if this is reasonable or practical to do.
-Normally all fields would be one or the other.
+Quando se trata de campos em uma struct, a primeira letra declara se o campo é
+acessível ao código fora do pacote em que está declarado. Nesse caso, Nome e ID estão acessíveis, 
+mas a senha não. É uma expressão idiomática separar campos exportados e não 
+exportados dessa maneira, se isso for razoável ou prático. Normalmente todos os 
+campos seriam um ou outro.
```
[the remark this answers is not in the document]
Comment (luciormoraes) [reply _content/tour/por/exporting.article:90]: Yes, absolutely.

**Pull request merged**